[Review] 2차 코드리뷰 풀리퀘스트입니다

Creamcheesepie opened the pull request:
## 멘토님 리뷰 잘 부탁 드립니다.

otr995 committed:
Merge branch 'main' of https://github.com/prgrms-be-devcourse/NBE7-9-2-Team9 into feat/bookmark

Creamcheesepie committed:
[feat/plan-front]: 계획 상세 부분에 카테고리별 상세 추천 추가

Creamcheesepie committed:
feat/plan,planDetail : 로그인 헤더 사용으로 변경

whyin committed:
feat: 프론트엔드 회원 구현

junseokPP committed:
Merge pull request #90 from whyin/feat/front-member

[feat] 프론트엔드 회원 관련 구현

Creamcheesepie committed:
Merge branch 'main' of https://github.com/prgrms-be-devcourse/NBE7-9-2-Team9 into feat/plan-front

Creamcheesepie committed:
feat/plan-front: apiResponse 사용해 응답에 헤더 적용, plan 요청시 스프링 인가 필요하게 sicurityConfig 설정

tobbot16 committed:
feat: 리뷰 추천 관련 백엔드 기능 추가 (frontend 제외)

junseokPP committed:
관리자 대시보드 페이지 구현

- AdminDashboardPage 컴포넌트 추가 (회원/여행지 선택 화면)
- 관리자 라우팅 구조 개선 (대시보드 → 카테고리 목록)
- 회원 관리와 여행지 관리 분리된 선택 화면
- 반응형 디자인 및 호버 효과 적용
- 팀원이 구현한 회원 관리 기능 통합

Creamcheesepie committed:
fix/plan: 누락된 항목 보충 커밋

junseokPP committed:
Merge pull request #91 from prgrms-be-devcourse/feat/plan-front

[Feat/plan-front] 로그인 후 인증정보 사용 , Security Config에 plan 도메인 추가

junseokPP committed:
Merge pull request #92 from prgrms-be-devcourse/feat/review-backend-only

[feat(review)] : 리뷰 추천 관련 백엔드 기능 추가 (frontend 제외)

junseokPP committed:
Merge pull request #93 from prgrms-be-devcourse/feat/admin-frontend

[feat] 관리자 대시보드 페이지 구현

whyin committed:
fix: 토큰 인자값 추가

junseokPP committed:
Merge pull request #94 from whyin/fix/auth

[fix] 토큰 인자값 추가

junseokPP committed:
UI 크기 최적화: 카테고리 박스 크기 줄이고 여행지 목록 영역 높이 늘리기

whyin committed:
feat: 관리자 화면 구현 및 회원 부분 수정

whyin committed:
fix(jwt): 토큰 에러코드 수정

whyin committed:
fix(jwt): 토큰 에러코드 수정

whyin committed:
fix(member): valid 추가

whyin committed:
fix(jwt): 주석 수정

Creamcheesepie committed:
Merge pull request #98 from whyin/fix/admin

[Fix/admin] 에러코드 401로 수정

Creamcheesepie committed:
fix/plan: 오늘의 계획 부븐 수정중

Creamcheesepie committed:
Merge branch 'main' of https://github.com/prgrms-be-devcourse/NBE7-9-2-Team9 into feat/plan-front

whyin committed:
fix(member): member 관련 수정

tobbot16 committed:
feat(review): 내가 작성한 리뷰가 보이도록 구현

whyin committed:
fix: controller 리턴값 응답 수정

otr995 committed:
Merge branch 'main' of https://github.com/prgrms-be-devcourse/NBE7-9-2-Team9 into feat/bookmark

whyin committed:
fix: 관리자 삭제 방지 로직 추가

whyin committed:
fix: 디티오 수정

whyin committed:
fix: 디티오 수정

whyin committed:
fix(member): 회원 상태 추가

tobbot16 committed:
Merge branch 'main' into feat/review-backend-only

junseokPP committed:
Merge pull request #96 from whyin/feat/front-member

[feat] 프론트엔드 관리자 화면 구현 및 회원 부분 수정

junseokPP committed:
Merge pull request #97 from prgrms-be-devcourse/feat/admin-frontend

UI 크기 최적화: 카테고리 박스 크기 줄이고 여행지 목록 영역 높이 늘리기

Creamcheesepie committed:
fix/planDetail: 계획 상세 삭제 버그 수정

whyin committed:
fix(jwt): 에러 코드 수정

junseokPP committed:
Merge pull request #101 from whyin/fix/member

[Fix/member] 회원부분 코드 수정

otr995 committed:
Merge branch 'main' of https://github.com/prgrms-be-devcourse/NBE7-9-2-Team9 into feat/bookmark

Creamcheesepie committed:
feat/plan-front: 전면페이지 약간 수정 및 상세 페이지 추가

Creamcheesepie committed:
Merge branch 'main' of https://github.com/prgrms-be-devcourse/NBE7-9-2-Team9 into feat/plan-front

whyin committed:
feat: 회원 상태 추가

whyin committed:
fix(jwt): 엑세스 토큰으로만 인증/ 리프레시토큰은 엑세스 토큰 발급용

tobbot16 committed:
feat(review): 수정하는 부분 수정

tobbot16 committed:
feat(review): 프론트 작업물

otr995 committed:
feat(bookmark): 북마크 프론트 구현

junseokPP committed:
Merge pull request #103 from prgrms-be-devcourse/feat/plan-front

[Feat/plan front] 계획 상세 삭제 처리 수정, 프론트 다듬기

junseokPP committed:
Merge pull request #104 from whyin/feat/front-admin

[feat/front-admin] 회원 상태 추가

junseokPP committed:
Merge pull request #105 from prgrms-be-devcourse/feat/review-backend-only

[Feat/review] 리뷰 수정부분 및 내가 쓴 리뷰 수정, 프론트 구현

junseokPP committed:
Merge branch 'main' into feat/bookmark

junseokPP committed:
Merge pull request #106 from prgrms-be-devcourse/feat/bookmark

[Feat/bookmark] 북마크 프론트 구현

Creamcheesepie committed:
fix/plan-front: 파일명 불일치 문제 수정

Creamcheesepie committed:
Merge pull request #107 from prgrms-be-devcourse/feat/plan-front

fix/plan-front: 파일명 불일치 문제 수정

Creamcheesepie committed:
fix/review : App.js 수정

Creamcheesepie committed:
Merge pull request #108 from prgrms-be-devcourse/feat/plan-front

fix/review : App.js 수정

Creamcheesepie committed:
fix/Plan-front : 파일명 수정 반영

Creamcheesepie committed:
feat/plan-front : 파일명 맞춰서 import 수정

Creamcheesepie committed:
Merge pull request #109 from prgrms-be-devcourse/feat/plan-front

Feat/plan front

tobbot16 committed:
feat(review): 리뷰 페이지 navigate 수정

Creamcheesepie committed:
feat/plan-front: 여행지 카테고리에 북마크 부분 추가 및 오류 요소 수정

tobbot16 committed:
feat(review): 카테고리별 전체 장소 별점순

whyin committed:
fix(security): api 토큰 없이 호출 불가능

otr995 committed:
feat(bookmark/front) : 북마크 프론트 apirequest 수정 및 추가 버튼 추가

whyin committed:
fix(member): 탈퇴한 회원 로그인 성공 오류 수정

Creamcheesepie committed:
Merge pull request #112 from prgrms-be-devcourse/feat/plan-front

[feat/plan-front] 여행지 카테고리에 북마크 부분 추가 및 오류 요소 수정

junseokPP committed:
Merge pull request #111 from prgrms-be-devcourse/feat/review-backend-only

[Feat/review]: 리뷰 페이지 navigate 수정, 카테코리별 별점순 여행지

junseokPP committed:
Merge pull request #113 from whyin/fix/security

[fix/security] api 토큰 없이 호출 불가능

junseokPP committed:
Merge pull request #114 from prgrms-be-devcourse/front/bookmark

[Feat(bookmark/front)] : 북마크 프론트 apirequest 수정 및 추가 버튼 추가

Creamcheesepie committed:
docs :스웨거 문서화 틀 만들기

Creamcheesepie committed:
merge: 충돌 내용 수정 수 커밋

whyin committed:
Update README.md

whyin committed:
Revise project overview in README

Updated project overview section to include future plans.

whyin committed:
Update README.md

whyin committed:
Update team member roles and responsibilities

Creamcheesepie committed:
Merge branch 'main' of https://github.com/prgrms-be-devcourse/NBE7-9-2-Team9 into feat/plan-front

Creamcheesepie committed:
doc/plan : 계획 스웨거 명세 추가

junseokPP committed:
feat: 로그인 시스템 개선 및 관리자 기능 연결 완료

- 홈화면을 로그인화면으로 변경
- 로그인 후 역할에 따라 /admin 또는 /user로 리다이렉트
- 회원가입 버튼 스타일 개선 (그린 계열, 크기 조정)
- 관리자 회원관리 기능 연결 (/admin/members)
- API 엔드포인트 수정 (관리자용: /api/admin/*, 사용자용: /api/*)
- 모든 페이지에 뒤로가기 버튼 및 로그아웃 버튼 추가
- 사용자 여행지 기능 복원 (카테고리 목록, 검색, 북마크)
- 일관된 하늘색 테마 및 글래스모피즘 디자인 적용

otr995 committed:
feat(bookmark) : 북마크 계획 상세 추가하기 구현 및 api.js 토스트 추가

Account-4-Review commented on backend/src/main/java/com/backend/domain/review/service/ReviewService.java:
```diff
@@ -169,4 +187,121 @@ public boolean validWithReviewId(Long memberId, Long reviewId){
         return review.getMember().getId().equals(member.getId());
     }
 
-}
```
```java
	public List<RecommendResponse> recommend(RecommendationType recommendationType) {
		Category category = categoryRepository.findByName(recommendationType.name()).orElseThrow(
			() -> new BusinessException(ErrorCode.NOT_FOUND_CATEGORY)
		);

		List<Map.Entry<Long, Double>> sortedList = getAllPlacesAndCalculate(category);

		return sortedList.stream()
			.limit(5)
			.map(placeInfo -> RecommendResponse.from(getPlaceEntity(placeInfo.getKey()), placeInfo.getValue()))
			.toList();
	}
```

Creamcheesepie committed:
doc : 문서 설정에 보안설정 추가

Creamcheesepie committed:
Merge branch 'main' into feat/bookmark

Creamcheesepie committed:
Merge pull request #115 from prgrms-be-devcourse/feat/plan-front

[doc] 스프링 문서화 추가

Creamcheesepie committed:
Merge pull request #116 from prgrms-be-devcourse/feat/bookmark

[feat/bookmark] : 북마크 계획 상세 추가하기 구현 및 api.js 토스트 추가

junseokPP committed:
Merge main into feat/admin-frontend - resolve conflict in BookmarkApp.jsx

junseokPP committed:
fix: 버튼 클릭 이벤트 문제 수정

- UserApp 배경 레이어(::before)에 pointer-events: none 추가
- 모든 페이지에 z-index 적용하여 버튼 클릭 가능하도록 수정
- 여행계획 페이지에서 window.location.href -> navigate()로 변경
- 북마크 페이지에 CSS 파일 import 추가
- 디버깅용 console.log 추가

junseokPP committed:
feat: 페이지 스타일 개선

- 리뷰 페이지 컬러풀한 핑크/퍼플 그라데이션 적용
- 북마크 페이지 컬러풀한 그린/블루 그라데이션 적용
- 관리자 회원관리 페이지 무채색으로 단정하게 변경
- 여행계획 페이지 배경색 통일
- 여행계획 작성 페이지 취소 버튼 클릭시 /user/plan으로 이동하도록 수정

junseokPP committed:
Merge pull request #117 from prgrms-be-devcourse/feat/admin-frontend

Feat/admin frontend

whyin committed:
Add ERD image to README

junseokPP committed:
refactor(yml) : dev->prod로 db교체

junseokPP committed:
Merge pull request #118 from prgrms-be-devcourse/feat/admin-frontend

refactor(yml) : dev->prod로 db교체

whyin committed:
Update README.md

otr995 committed:
Update team member responsibilities in README

tobbot16 committed:
Update team member responsibilities in README

junseokPP committed:
Update README.md

Creamcheesepie committed:
fix:git ignore 복구

junseokPP committed:
refac

junseokPP committed:
오류

junseokPP committed:
준석오류

junseokPP committed:
Merge pull request #119 from prgrms-be-devcourse/feat/plan

fix:git ignore 복구

junseokPP committed:
Clean up tracked build and cache files

Creamcheesepie committed:
.idea 추적제외

Creamcheesepie committed:
패키지 명 수정

tobbot16 committed:
feat : 리뷰에 한줄 추가 기능 달기

tobbot16 committed:
feat: 가중치 설정값 세팅

tobbot16 committed:
feat: admin부분 review - 인증부분은 미구현

tobbot16 committed:
feat: 캐시 및 리뷰 최적화, Recommend 엔티티, repository 생성

tobbot16 committed:
Revert "feat: 캐시 및 리뷰 최적화, Recommend 엔티티, repository 생성"

This reverts commit fa2ebe250c613482c47f13b59f446eff0e82b96d.

tobbot16 committed:
B
Reapply "feat: 캐시 및 리뷰 최적화, Recommend 엔티티, repository 생성"

This reverts commit 01f319f7c58bf50c99856586e6f01dbe86fcdb18.
